activemq-broker/src/main/java/org/apache/activemq/broker/BrokerService.java

```diff
@@ -1672,11 +1677,28 @@ public boolean isShutdownOnMasterFailure() {
     }
 
     /**
+     * This value shutdownOnMasterFailure is deprecated and will be removed in a future release. Use shutdownOnActiveFailure instead.
+     * @deprecated
      * @param shutdownOnMasterFailure
-     *            The shutdownOnMasterFailure to set.
      */
     public void setShutdownOnMasterFailure(boolean shutdownOnMasterFailure) {
         this.shutdownOnMasterFailure = shutdownOnMasterFailure;
+        this.shutdownOnActiveFailure = shutdownOnMasterFailure;
+        LOG.warn("shutdownOnMasterFailure is deprecated and will be removed in a future release. Use shutdownOnActiveFailure instead.");
```

Comment:
Maybe worth phrasing, `"configuration 'shutdownOnMasterFailure' is deprecated and will be removed in a future release. Use 'shutdownOnActiveFailure' instead"` ?

Reply:
(similar usage of the word configuration across the board, to make it clear the behaviour is unchanged).

Reply:
How's this? 

```
    /**
     * Configuration `passiveSlave` is deprecated and will be removed in a future release. Use `standbyInstance` instead.
     * @deprecated AMQ-7514 in the move to inclusive nomenclature
     * @return the passiveSlave
     */
    @Deprecated
    public boolean isPassiveSlave() {
        return this.passiveSlave;
    }
```